specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1474,6 +1474,10 @@
           "type": "string",
           "description": "Version of orchestrator specified when creating the managed cluster."
         },
+        "upgrade": {
```

Comment:
so my main concern is this is returned in two different apis but will only ever be populate in the agentpool api not the managed cluster one. Do we need to duplicate (or extand with allof) this object and make agentpool api return the extended while managed cluster only resturns the original. This will effect Frank Gao's pending request to.